config/environments/test.rb

```diff
@@ -36,10 +36,6 @@
 
   config.middleware.use RackSessionAccess::Middleware
 
-  # Disable lograge when computing coverage and not in CircleCI, where lograge is required.
-  # This enables scanning for view test coverage with `rake test:scan_log_for_view_coverage`
-  config.lograge.enabled = !ENV['COVERAGE'] || ENV['CI']
```

Comment:
Just to confirm, should we still be enabling this given the other half of the condition?

```suggestion
  # Disable lograge when computing coverage, where lograge is required.
  # This enables scanning for view test coverage with `rake test:scan_log_for_view_coverage`
  config.lograge.enabled = !ENV['COVERAGE']
````

Reply:
It didn't impact the current coverage calculation (https://gitlab.login.gov/lg/identity-idp/-/jobs/51849), so it seems ok for now.